src/Servers.jl

```diff
@@ -1,101 +1,74 @@
+"""
+The `HTTP.Servers` module provides server-side http functionality in pure Julia.
+
+The main entry point is `HTTP.listen(handler, host, port; kw...)` which takes a `handler` argument (see `?HTTP.Handlers`),
+a `host` and `port` and optional keyword arguments. For full details, see `?HTTP.listen`.
+"""
 module Servers
 
+export listen, handle, Handler, RequestHandler, StreamHandler
```

Comment:
I see that the `RequeatHandler` type is used by `Routers.jl`, so I guess it has to be exported from here.
However, `StreamHandler` is only used within this file (and does not appear in `../test/*`. No need to export it if it isn't used anywhere.

Is there a use case for a custom StreamHandler subtype?

Reply:
I don't see any reason to restrict it.